projects/design/src/app/view-component/activities/associated-skills/edit-mode/edit-mode.component.ts

```diff
@@ -209,7 +208,7 @@ export class AssociatedSkillsEditModeComponent implements OnInit {
       title: node.title,
       type: 'skill'
     })
-    this.router.navigate([`/task/${node.ID}`], {
+    this.router.navigate([`/design/task/${node.ID}`], {
```

Comment:
I suppose a component should not know about the prefix of the route... Not necessarily to be fixed right now, but just looks strange